bokehjs/src/coffee/models/annotations/tooltip.coffee

```diff
@@ -2,6 +2,16 @@ import {Annotation, AnnotationView} from "./annotation"
 import {div, show, hide, empty} from "core/dom"
 import * as p from "core/properties"
 
+export compute_side = (attachment, sx, sy, hcenter, vcenter) ->
+  switch attachment
+    when "horizontal"
+      side = if sx < hcenter then 'right' else 'left'
+    when "vertical"
+      side = if sy > vcenter then 'below' else 'above'
```

Comment:
This doesn't look correct to me, but maybe I'm misunderstanding what `above` and `below` actually mean here.

Reply:
It wasn't, I just pushed the real fix